### ISSUE TITLE
Browser Sync does not load - white screen with Webpack / Wordpress 

### ISSUE THREAD

**Issue by irenehardy** [BrowserSync/browser-sync#2013]
### Issue details

Running browser-sync via webpack for a Wordpress plugin. Everything compiles fine, but the proxied browser-sync window just opens a white screen that hangs on loading infinitely. No source code, no console information. Happens both via webpack, and in the command line directly calling browser-sync with the proxy url. 

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.27.11  ]
- Node        [  18.13.0  ]
- Npm         [  8.19.3  ]

### Affected platforms

- [ ] linux
- [ ] windows
- [ x ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [ ] Gulp
- [ ] Grunt
- [ x ] CLI
- [ x ] Webpack

### If CLI, please paste the entire command below

browser-sync https://[[redacted]].test

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

Webpack config - 

const MiniCssExtractPlugin = require('mini-css-extract-plugin');
const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
var path = require('path');

// change these variables to fit your project
const jsPath= './public/js';
const cssPath = './public/scss';
<img width="1306" alt="Screen Shot 2023-02-16 at 9 04 37 PM" src="https://user-images.githubusercontent.com/24785387/219530846-eb9cd7ee-fb38-4d2e-92aa-165ddb0a6abe.png">

const outputPath = './public/dist';
const localDomain = 'https://[[redacted]].test/';
const entryPoints = {
  // 'app' is the output name, people commonly use 'bundle'
  // you can have more than 1 entry point
  'app': jsPath + '/scripts.js',
};

module.exports = {
  entry: entryPoints,
  output: {
    path: path.resolve(__dirname, outputPath),
    filename: '[name].js',
  },
  plugins: [
    new MiniCssExtractPlugin({
      filename: '[name].css',
    }),

    // Uncomment this if you want to use CSS Live reload
    new BrowserSyncPlugin({
        proxy:  {
            target: localDomain,
            },
        host: '127.0.0.1',
        files: [ outputPath + '/*.css' ],
        injectCss: true
    }, 
    { reload: false, }),
  ],
  module: {
    rules: [
      {
        test: /\.s?[c]ss$/i,
        use: [
          MiniCssExtractPlugin.loader,
          'css-loader',
          'sass-loader',
        ],
      },
      {
        test: /\.sass$/i,
        use: [
          MiniCssExtractPlugin.loader,
          'css-loader',
          {
            loader: 'sass-loader',
            options: {
              sassOptions: { indentedSyntax: true },
            },
          },
        ],
      },
      {
        test: /\.(jpg|jpeg|png|gif|woff|woff2|eot|ttf|svg)$/i,
        use: 'url-loader?limit=1024',
      },
    ]
  },
};

**Comment by DeoThemes** [comment]
I'm having the same issue. Have you managed to resolve?

**Comment by irenehardy** [comment]
Yes and no. LocalWP advised against using SSL with browser sync, so I was able to get it working with a non-secure local version of the site. I still would love to resolve it, because it used to be no trouble using SSL in proxy mode, but works well enough for the time being.

**Comment by jm-byteberry** [comment]
Same issue here also using macOS. It used to work for me until now. I guess something I've recently updated broke the compatibility.
